src/App.php

```diff
@@ -234,48 +232,6 @@ public function processConsole(array $serverParams): void
 		(\Friendica\Core\Console::create($this->container, $argv))->execute();
 	}
 
-	/**
-	 * @internal
-	 */
-	public function processEjabberd(array $serverParams): void
-	{
-		$this->setupContainerForAddons();
-
-		$this->setupLogChannel(LogChannel::AUTH_JABBERED);
```

Comment:
Please add support for `LogChannel::AUTH_JABBERED` in `App::determineLogChannel()`

Reply:
Done :D